doc/riverctl.1.scd

```diff
@@ -389,6 +389,10 @@ matches everything while _\*\*_ and the empty string are invalid.
 	"grp:ctrl_space_toggle"). See *xkeyboard-config*(7) for possible values and
 	more information.
 
+*keyboard-layout-file* _path_
+	Sets the XKB layout for all keyboards from a definition file at the provided
```

Comment:
Consistent tense.
```suggestion
	Set the XKB layout for all keyboards from a definition file at the provided
```
It would also be nice to reference some documentation on the format used by xkbcommon if such documentation exists (ideally as a man page). Is this the same `*xkeyboard-config*(7)` we mention above?